python_modules/libraries/dagster-deltalake/dagster_deltalake/handler.py

```diff
@@ -55,28 +55,56 @@ def handle_output(
         connection: TableConnection,
     ):
         """Stores pyarrow types in Delta table."""
+        assert context.metadata is not None
+        assert context.resource_config is not None
         reader, delta_params = self.to_arrow(obj=obj)
         delta_schema = Schema.from_pyarrow(reader.schema)
 
+        engine = context.resource_config.get("writer_engine")
+        save_mode = context.metadata.get("mode")
+        main_save_mode = context.resource_config.get("mode")
+        main_custom_metadata = context.resource_config.get("custom_metadata")
+        overwrite_schema = context.resource_config.get("overwrite_schema")
+        writerprops = context.resource_config.get("writer_properties")
+
+        if save_mode is not None:
+            context.log.info(
+                "IO manager mode overridden with the asset metadata mode, %s -> %s",
+                main_save_mode,
+                save_mode,
+            )
+            main_save_mode = save_mode
+        context.log.info("Writing with mode: %s", main_save_mode)
```

Comment:
are these really info worthy, or should those be `debug`? passing some config via metadata is the "official" way, so this is just expected bahviour?

Reply:
Started using dagster last Saturday so I don't know all the concepts yet of what way is correct or not : P

I'll change it to DEBUG instead